extension/src/repository/data/index.ts

```diff
@@ -141,12 +141,15 @@ export class RepositoryData extends DeferredDisposable {
     const gitRoot = await getGitRepositoryRoot(this.dvcRoot)
 
     this.dispose.track(
-      createFileSystemWatcher(join(this.dvcRoot, '**'), (path: string) => {
-        if (isExcluded(this.dvcRoot, path)) {
-          return
+      createFileSystemWatcher(
+        new RelativePattern(this.dvcRoot, '**/*.*'),
```

Comment:
Will this have issues with files that don't have an extension? Would `**/*` work?